src/models/runner.rs

```diff
@@ -26,3 +34,95 @@ pub struct Runner {
     pub running_level: String,
     pub donation: String,
 }
+
+impl<'a> From<&'a Info> for NewRunner<'a> {
+    fn from(info: &'a Info) -> Self {
+        let conn = &mut establish_connection();
```

Comment:
One concern here is that a new connection is established. Might be worthwhile to reuse the one initialized in `join::register`